src/sales/orders-search.md

```diff
@@ -28,14 +28,19 @@ _Search Results_
 
 |Filter|Description|
 |--- |--- |
+|Purchase Date|Filters the search based on the date purchased. To find orders within a range of dates, enter both the **from** and **to** dates.|
 |ID|Filters the search based on order ID.|
+|Grand Total (Base)|Filters the search based on the Grand Total of each order.|
```

Comment:
am i missing something?  these descriptions seem the same.  is Grand Total (Purchased) the total number of items in the order or the total price of the items in the order?